SAST/gitlab-sast-cloud.yml

```diff
@@ -0,0 +1,20 @@
+# add SAST_ORGANIZATION and SASTCLI_TOKEN as an environment variable - https://gitlab.com/help/ci/variables/README
+default:
+  image: ubuntu:latest
+
+stages:
+  - scan
+
+ws_scan:
+  stage: scan
+  variables:
+    SAST_ORGANIZATION: $SAST_ORGANIZATION
+    SASTCLI_TOKEN: $SASTCLI_TOKEN
+    SAST_SERVER: "https://sast-demo.whitesourcesoftware.com"
+  script:
+    - echo "Downloading WS SAST Tool"
+    - curl -LJO https://downloads-demo.whitesourcesoftware.com/sast-cli/linux/wscli && chmod +x wscli
+    - echo "Run WS SAST"
+    - ./wscli --dir ./ --name=GL_$CI_PROJECT_NAME -app GitLab_Pipeline
```

Comment:
-name should be repo name + branch & -app should be CI_PROJECT_NAME

Reply:
How is this?

Ref: https://docs.gitlab.com/ee/ci/variables/predefined_variables.html